e2e-tests/portal/cypress/e2e/pages/utilisation-report-service/record-corrections/pendingCorrections.js

```diff
@@ -0,0 +1,16 @@
+const page = {
+  table: () => cy.get('[data-cy="pending-corrections-table"]'),
+  rows: () => cy.get(`[data-cy="pending-corrections-table"] tr`),
+  row: (index) => ({
+    facilityId: () => cy.get(`[data-cy="pending-corrections-table"] tr:nth-child(${index}) [data-cy="pending-corrections-row--facility-id"]`),
```

Comment:
could each row not use a correctionId so we don't need to find by row index?

Reply:
I guess we could, but I thought this was pretty nice and will be useful when we are doing e2e tests for the full complete journey where we won't know what the id is.

Reply:
lets leave it for now.  I have something in my PR to find the record correction id so we can refactor later once my PR is in